### ISSUE TITLE
Availability of 0.34.24 on maven central

### ISSUE THREAD

**Issue by m-manseer** [vsch/flexmark-java#264]
Great job on this project @vsch and others! I'm really looking forward to trying the new `flexmark-ext-gitlab` extension, but can't find version **0.34.24** on [maven central](https://mvnrepository.com/artifact/com.vladsch.flexmark/flexmark).

And idea when it will become available? Keep up the good work!

**Comment by vsch** [comment]
@m-manseer, I deployed it and released 5 days ago. Just checked and for some reason it is lost in aether. I did not check since maven central usually takes a few hours to update but rarely fails. Thank you for letting me know.

I am re-deploying and will update maven central again.

**Comment by m-manseer** [comment]
Excellent news, thank you.

**Comment by vsch** [comment]
@m-manseer, the badge is showing 0.34.24 but maven central is still showing 0.34.22 as latest. However, IntelliJ IDE is already seeing 0.34.24 as available.

I am guessing it will take a bit more time for the update to propagate to maven central search.

**Comment by m-manseer** [comment]
Thanks - did you resubmit `core` only or also `all`?
I'm using `all` in my .pom file ([this one here](https://mvnrepository.com/artifact/com.vladsch.flexmark/flexmark-all)).

```
<dependency>
    <groupId>com.vladsch.flexmark</groupId>
    <artifactId>flexmark-all</artifactId>
    <version>0.34.24</version>
</dependency>
```

Eclipse doesn't find it yet, but I'm sure it will come through eventually. I'll let you know once it works.

**Comment by vsch** [comment]
@m-manseer, I just checked and maven central search reports the latest as 0.34.24 now.

**Comment by m-manseer** [comment]
Indeed, I can see it on maven central. For some strange reason Eclipse won't find it yet. I'll give it another day.

> Missing artifact com.vladsch.flexmark:flexmark-all:jar:0.34.24

**Comment by vsch** [comment]
@m-manseer, I don't use Eclipse so cannot be sure, but is there an "update repository" functionality to update maven local cache?
